PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp

```diff
@@ -53,7 +53,7 @@ wstring getTextFromCombo(HWND hCombo)
 	auto str = std::make_unique<wchar_t[]>(strSize);
 	std::fill_n(str.get(), strSize, L'\0');
 
-	::SendMessage(hCombo, WM_GETTEXT, FINDREPLACE_MAXLENGTH - 1, reinterpret_cast<LPARAM>(str.get()));
+	::SendMessage(hCombo, WM_GETTEXT, FINDREPLACE_MAXLENGTH, reinterpret_cast<LPARAM>(str.get()));

```

Comment:
The API for WM_GETTEXT, [here](https://learn.microsoft.com/en-us/windows/win32/winmsg/wm-gettext) says that "wParam is the maximum number of characters to be copied, including the terminating null character."  Thus, we should not subtract 1 from FINDREPLACE_MAXLENGTH here; doing so limits the number of characters copied to FINDREPLACE_MAXLENGTH - 2 which is not what we want.